frontend/src/components/tournament/ReorderTeams.js

```diff
@@ -0,0 +1,92 @@
+import { useDragDropContext } from "@thisbeyond/solid-dnd";
+import {
+  DragDropProvider,
+  DragDropSensors,
+  DragOverlay,
+  SortableProvider,
+  createSortable,
+  closestCenter
+} from "@thisbeyond/solid-dnd";
+import { createSignal, For } from "solid-js";
+
+const TeamInfo = props => (
+  <div class="flex flex-row p-2">
+    <div class="basis-3/4 text-start">{props.teamName}</div>
+    <div class="basis-1/4 dark:bg-gray-600/50 bg-gray-100 text-center rounded-xl">
+      {props.teamId}
+    </div>
+  </div>
+);
+
+const Sortable = props => {
+  const sortable = createSortable(props.teamId);
+  const [state] = useDragDropContext();
+  return (
+    <div
+      use:sortable
+      class="sortable text-gray-500 dark:text-gray-400 m-2 rounded-lg bg-gray-200 border-gray-300 border-b dark:bg-gray-800 dark:border-gray-700"
+      classList={{
+        "opacity-25": sortable.isActiveDraggable,
+        "transition-transform": !!state.active.draggable
+      }}
+    >
+      <TeamInfo teamId={props.teamId} teamName={props.teamsMap[props.teamId]} />
```

Comment:
Having the team name + id rendered as separate elements, and the seeds as separate ones causes issues when the team name wraps around into two different lines. 

![image](https://github.com/india-ultimate/hub/assets/315678/b37f5276-418a-45cd-a411-a3dc6856770a)

Reply:
Yes, i did see this when i was messing around with the widths of the team name div.
- Removing the team name would give some space
- My thought process for rendering seeds separately was that, teams are being assigned seeds. Its really the teams moving, and not the seed values in itself.

I think a CSS fix is sufficient, i'll try to increase size of the team size div. what do you think ?

Reply:
I agree with you on moving the teams around, but CSS fixes seem hacky to me; It may just be me. What if a team has a really long name - for instance `Learning to Fly` or `Breakside (Thatte Idli Kaal Soup)`. But, may be I'm overthinking this. Using the full width on smaller screens, and removing the team ID may be sufficient.

Reply:
yeah, i checked after removing teamID and full width on smaller screens. The issue might go away with this, although it is hacky. I've attached a video below, maybe you can check and see if that looks good ?